hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/offlineImageViewer/PBImageDelimitedTextWriter.java

```diff
@@ -146,14 +168,21 @@ public String build() {
   PBImageDelimitedTextWriter(PrintStream out, String delimiter,
                              String tempPath, boolean printStoragePolicy)
       throws IOException {
-    this(out, delimiter, tempPath, printStoragePolicy, 1, "-");
+    this(out, delimiter, tempPath, printStoragePolicy, 1, "-", null, false);
   }
 
   PBImageDelimitedTextWriter(PrintStream out, String delimiter,
-      String tempPath, boolean printStoragePolicy, int threads,
-      String parallelOut) throws IOException {
+                             String tempPath, boolean printStoragePolicy,
+                             int threads, String parallelOut,
+                             Configuration conf, boolean printECPolicy)
```

Comment:
I think it may be better to define this method as the following.
```
  PBImageDelimitedTextWriter(PrintStream out, String delimiter,
                             String tempPath, boolean printStoragePolicy,
                             boolean printECPolicy, int threads, String parallelOut,
                             Configuration conf)
```